sources/packages/web/src/components/institutions/request-a-change/CompletedOfferingChangeSummary.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <tab-container>
+    <body-header-container>
+      <template #header>
+        <body-header title="Applications" :recordsCount="applications?.count">
+          <template #subtitle>Outcome of the requested change</template>
+          <template #actions>
+            <v-text-field
+              density="compact"
+              label="Search name or application #"
+              variant="outlined"
+              v-model="searchCriteria"
+              @keyup.enter="searchApplicationOfferingChangeRecords"
+              prepend-inner-icon="mdi-magnify"
+              hide-details="auto"
+            >
+            </v-text-field>
+          </template>
+        </body-header>
+        <content-group>
+          <toggle-content :toggled="!applications?.count">
+            <v-data-table-server
+              :headers="CompletedOfferingChangeSummaryHeaders"
+              :items="applications?.results"
+              :items-length="applications?.count"
+              :loading="loading"
+              :items-per-page="DEFAULT_PAGE_LIMIT"
+              @update:options="paginationAndSortEvent"
+            >
+              <template #[`item.dateCompleted`]="{ item }">
+                {{ dateOnlyLongString(item.columns.dateCompleted) }}
+              </template>
+              <template #[`item.fullName`]="{ item }">
```

Comment:
Please update the Name and Application # sorting arrows as per figma and please fix the alignment of the arrows too. 

Screenshot:

![image](https://github.com/bcgov/SIMS/assets/7859295/3ecc5d63-4191-401e-a178-37441e19b746)

Reply:
It was decided to follow the bootstrap default behavior for sort. it will appear on the mouse hover

Reply:
I'm not sure if we are going to customize it @sh16011993 . The idea is to use as it is  unless it does not do the minimum necessary as we are working on an MVP. That's my understanding. @hellolynn-tbtb can say if that's ok or not.